roles/redpanda_broker/tasks/install-rp-deb.yml

```diff
@@ -4,21 +4,46 @@
     using_gcp: true
   when: ansible_system_vendor == 'Google'
 
-- name: Set fact for package name
+- name: Set fact for package names
   ansible.builtin.set_fact:
-    redpanda_package_name: "redpanda{{ '' if redpanda_version=='latest' else ('=' if ansible_os_family == 'Debian' else '-') + redpanda_version }}"
+    redpanda_version_suffix: "{{ '' if redpanda_version == 'latest' else ('=' if ansible_os_family == 'Debian' else '-') + redpanda_version }}"
+    needs_split_packages: >-
+      {{ redpanda_version == 'latest' or 
+         (redpanda_version is version('24.2.0', '>=')) or
```

Comment:
Can this compare `>=` if the version includes a suffix? e.g our `24.2.1-rc1`?

Reply:
@r-vasquez dming you but I tested with some weird versions and it did ok 😅